doc/book/controllers.md

```diff
@@ -280,9 +280,9 @@ extending class.
 ## AbstractConsoleController
 
 > To use  it, you need to install zend-mvc-console via
```

Comment:
Something more is missing:

- zend-mvc-console is needed in version 3
- the component must be added as a module

> For version 3 the integration component [zend-mvc-console](https://docs.zendframework.com/zend-mvc-console/) must be installed. It can be done via  Composer:
> ```bash
> composer require zendframework/zend-mvc-console
> ```
> If you are not using the component installer, you will need to [add this component as a module](https://docs.zendframework.com/zend-mvc-console/intro/#manual-installation).